### ISSUE TITLE
Removing 1 more warning

### ISSUE THREAD

**Issue by oleedd1** [Ceiridge/Chrome-Developer-Mode-Extension-Warning-Patcher#85]
Please make possible enabling extensions not from the store.
![image](https://user-images.githubusercontent.com/36125949/203298834-c69d4b75-b8b1-4c71-8747-7a43edb925cb.png)

**Comment by oleedd1** [comment]
Yes, but better to remove this warning too. It is much more convenient. Please.

**Comment by oleedd1** [comment]
Yes, but better to remove this warning too. Usual installing is much more convenient. Please.

**Comment by Ceiridge** [comment]
I will maybe look into it in the next days

**Comment by oleedd1** [comment]
It uses `verified_contents.json` for validation and it is done offline. Non-store extensions don't have this file.

**Comment by Ceiridge** [comment]
I don't think I will want to fix this. I don't want this patcher to have potential to be abused to install automatically install extensions without knowledge, which is often done by annoying antiviruses for example with their CRX file. It should be sufficient for normal use cases to just use the extracted folder?

**Comment by Ceiridge** [comment]
This is what disables the extension in case of a verification failure by the way: https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/extensions/install_verifier.cc;l=299;drc=4d9f1c1280ee8474a61ff89260a81651c50b4fb6;bpv=0;bpt=1

**Comment by oleedd1** [comment]
But Chromium allows to install any extensions (at least allowed).
And Chrome shows an alert when you try to install, for example:
![image](https://user-images.githubusercontent.com/118817520/204129196-1e442d65-4b78-4bb3-8c59-84072cbb7a1b.png)
And installing from url is already disabled, as I see.
Antiviruses etc. use store extensions. And it doesn't stop them to install automatically.
Anyway, it's enough for me if you just tell how to fix this in assembler.

**Comment by Ceiridge** [comment]
It's not too easy to patch this, mostly because helpful functions, such as *ShouldEnforce* have been removed during compilation because of optimization. I think it's too much effort for something that isn't especially useful.
If you want to take a look at it yourself, I recommend debugging the main Chrome process with x64dbg, looking at the chrome.dll symbol (make sure to *loadsym* the pdb debug symbol file of chrome.dll, which can be downloaded here with symchk for instance: https://chromium-browser-symsrv.commondatastorage.googleapis.com/ )

**Comment by oleedd1** [comment]
It is very important and useful because "Developer mode" is for developers while usual installing is for all users. Usual people don't search about "developer mode" and will not find this. And Chrome uses another saving algorithm for extensions from folders, no folders created for them, so it is not the same. There may be other differences.
But the last version of Chromium doesn't show this warning, all extensions work there, I have just checked this. How can the Chromium source code help?

**Comment by Ceiridge** [comment]
I don't really see a reason to add this.

**Comment by oleedd1** [comment]
Then warning removing will be ultimate. For now, you remove 1 but leave the 2-nd. So patching is not complete. More users will find this pather. It is a Chrome limitation to annoy users and force them to expend the store. Please.

**Comment by Ceiridge** [comment]
I will definitely keep this mind and will possibly look further into this issue in 1-2 business decades. In the meantime I can recommend extracting the extension first and then loading the folder.
